back.py

```diff
@@ -39,10 +43,12 @@ def GetCPUStateTimes():
         timelist.append(singlist)
     return timelist
 
+
 def GetCPUUsagePercent():
     cpuprcnt = psutil.cpu_percent(percpu=True)
     return cpuprcnt
 
+
```

Comment:
@ShivamVerma5665, the functionality looks and works fine to me,just the following changes requested:
- delete the extra blank lines added in file ```back.py```
- revert the class name from ```container``` to ```cuntainer```

Reply:
True. As `container` is a reserved keyword for the CSS libraries used here , please stick to the `cuntainer` name.